canonical/canonicalwebteam.templateparser-backend
# WD-12280 Changes to Parse more pages

# Done
- Changes to parse more pages
- Parses for error pages as well

# QA
- Run the server 
- Make api call to `localhost:<PORT>/get-tree/canonical.com/main`
- Check if it returns the  required pages in tree structure

## Fixes [WD-12280](https://warthogs.atlassian.net/browse/WD-12280)



[WD-12280]: https://warthogs.atlassian.net/browse/WD-12280?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
